### PR TITLE
My advanced portfolio

### PR DESCRIPTION
**Pull request by monicah79**
- Updated my index.html
- Added more styling in style.css
- Followed figma design

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -43,4 +43,166 @@ <h1>
...
     </section>
 
-    </div>
\ No newline at end of file
+    </div>
+</div>
+<section class="sub-grid-display">
+<div class="sub-homepage">
+    <div class="sub-card-details">
+        <img src="Images/snaps-portfolio.png" alt="">
```
Comment (Owusu-Desmond) [inline index.html:51]: - [ ] Please you should not have the `alt` attribute empty for your images. You should add a descriptive `alt` attribute to your images. :pray:

File: index.html [hunk 2]
```diff
@@ -43,4 +43,166 @@ <h1>
...
+    <h2 class="sub-header">
+        Tonic
+    </h2>
+    <ul class="sub-titles">
+        <li>CANOPY</li>
+        <li><img src="Images/Counter.png" alt=""></li>
+        <li>Back End dev</li>
+        <li><img src="Images/Counter.png" alt=""></li>
```
Comment (Owusu-Desmond) [inline index.html:59]: - [ ] Please I suggest you should not have the `alt` attribute empty for your images. You should add a descriptive `alt` attribute to your images for accessibility reasons :pray:

File: index.html [hunk 3]
```diff
@@ -43,4 +43,166 @@ <h1>
...
+        <li>javaScript</li>
+    </ul>
+    <button class="sub-button">
+       See Project
+    </button>
+    </div>
+</div>
+<div class="sub-homepage">
+    <div class="sub-card-details">
+        <img src="Images/snaps-calender.png" alt="">
```
Comment (Owusu-Desmond) [inline index.html:77]: - [ ] Please I suggest you should not have the `alt` attribute empty for your images. You should add a descriptive `alt` attribute to your images for accessibility reasons :pray:

File: index.html [hunk 4]
```diff
@@ -43,4 +43,166 @@ <h1>
...
+    <h2 class="sub-header">
+    Multi-Post <br>
+        Stories
+    </h2>
+    <ul class="sub-titles">
+        <li>CANOPY</li>
+        <li><img src="Images/Counter.png" alt=""></li>
+        <li>Back End dev</li>
+        <li><img src="Images/Counter.png" alt=""></li>
```
Comment (Owusu-Desmond) [inline index.html:86]: - [ ] Please I suggest you should not have the `alt` attribute empty for your images. You should add a descriptive `alt` attribute to your images for accessibility reasons :pray:

File: index.html [hunk 5]
```diff
@@ -43,4 +43,166 @@ <h1>
...
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+    </p>
+    <ul class="sub-icons">
+        <li>css</li>
+        <li>html</li>
+        <li>javaScript</li>
+    </ul>
+    <button class="sub-button">
+       See Project
+    </button>
```
Comment (Owusu-Desmond) [inline index.html:99]: - [ ] Please make sure your see project button follows the button styling: Active, Hover and Pressed to make it interactive. :pray:
 
For instance, 
 
   - You can change the background color, increase the size of the button, and change the color of the text when the user hovers over the button. You can visit [this link](https://www.w3schools.com/css/css3_buttons.asp) to learn more about it. :pray:
  
   - You can give it a `cursor: pointer` property to change the cursor to a pointer when the user hovers over it. You can read more about it [here](https://www.w3schools.com/cssref/pr_class_cursor.asp) :pray:

Screenshot from 2022-10-28 15-23-58

File: index.html [hunk 6]
```diff
@@ -43,4 +43,166 @@ <h1>
...
```
Comment (Owusu-Desmond) [inline index.html:125]: - [ ] Please make sure your see project button follows the button styling: Active, Hover and Pressed to make it interactive. :pray:
 
For instance, 
 
   - You can change the background color, increase the size of the button, and change the color of the text when the user hovers over the button. You can visit [this link](https://www.w3schools.com/css/css3_buttons.asp) to learn more about it. :pray:
  
   - You can give it a `cursor: pointer` property to change the cursor to a pointer when the user hovers over it. You can read more about it [here](https://www.w3schools.com/cssref/pr_class_cursor.asp) :pray:

Screenshot from 2022-10-28 15-23-58

File: index.html [hunk 7]
```diff
@@ -43,4 +43,166 @@ <h1>
...
```
Comment (Owusu-Desmond) [inline index.html:151]: - [ ] Please make sure your see project button follows the button styling: Active, Hover and Pressed to make it interactive. :pray:
 
For instance, 
 
   - You can change the background color, increase the size of the button, and change the color of the text when the user hovers over the button. You can visit [this link](https://www.w3schools.com/css/css3_buttons.asp) to learn more about it. :pray:
  
   - You can give it a `cursor: pointer` property to change the cursor to a pointer when the user hovers over it. You can read more about it [here](https://www.w3schools.com/cssref/pr_class_cursor.asp) :pray:

Screenshot from 2022-10-28 15-23-58

File: index.html [hunk 8]
```diff
@@ -43,4 +43,166 @@ <h1>
...
+    <li>
+        <a href=""><img src="Images/twitter.svg" alt=""></a>
+    </li>
+    <li>
+        <a href=""><img src="Images/Angel-list.svg" alt=""></a>
+    </li>
+</ul>
+<button class="sub-button">
+    Get My Resume
+ </button>
```
Comment (Owusu-Desmond) [inline index.html:184]: - [ ] Please make sure your see project button follows the button styling: Active, Hover and Pressed to make it interactive. :pray:
 
For instance, 
 
   - You can change the background color, increase the size of the button, and change the color of the text when the user hovers over the button. You can visit [this link](https://www.w3schools.com/css/css3_buttons.asp) to learn more about it. :pray:
  
   - You can give it a `cursor: pointer` property to change the cursor to a pointer when the user hovers over it. You can read more about it [here](https://www.w3schools.com/cssref/pr_class_cursor.asp) :pray:

Screenshot from 2022-10-28 15-33-11

File: index.html [hunk 9]
```diff
@@ -43,4 +43,166 @@ <h1>
...
+
+</section>
+<section class="sub-language">
+    <div class="sub-column">
+        <li class="language-heading"> Languages
+        <img src="Images/arrow-down-icon/Normal Button/Tertiary/Icons/Union.svg" alt=""/>
+        </li>
```
Comment (Owusu-Desmond) [inline index.html:191]: - [ ] Please you should only use the `ul` to wrap your list items `li` in your project. But you have used the `div` tag to wrap the `li` tag in your project. You can read more about it [here](https://www.w3schools.com/html/html_lists.asp) :pray:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @monicah79  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the set up the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 10]
```diff
@@ -26,21 +26,183 @@ <h1>
...
+        <a href=""><img src="Images/Angel-list.svg" alt="Angel-list icon"></a>
+    </li>
+</ul>
+<button class="sub-button">
+    Get My Resume
+ </button>
+
+</section>
+<section class="sub-language">
+    <div class="sub-column">
```
Comment (OyePriscilla) [inline index.html:188]: - [ ] Please kindly wrap the list of your technology with `ul` as the first reviewer suggested. 🙏 `ul` has semantics, `div` is just an ordinary container.

File: index.html [hunk 11]
```diff
@@ -26,21 +26,183 @@ <h1>
...
+    </li>
+    <li>
+        <a href=""><img src="Images/twitter.svg" alt="twitter icon"></a>
+    </li>
+    <li>
+        <a href=""><img src="Images/Angel-list.svg" alt="Angel-list icon"></a>
+    </li>
+</ul>
+<button class="sub-button">
+    Get My Resume
```
Comment (OyePriscilla) [inline index.html:183]: - [ ] Kindly please ensure to add button styling as the first reviewer suggested. 🙏 

## Illustration of what you should do:
.button:hover {
  background-color: #color according to template; 
  color: white;
}

- [ ] Please do the same for active and pressed styling as well. :smile

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
## Hi @monicah79   😄 ,
Honestly, you've done a great work 🤝, but there are few issues to rectify. Be encouraged, you're almost there. 👍 


 
## (Highlights) Good Points: 
* No linter errors. :heavy_check_mark:
* Followed Github flow :heavy_check_mark:
* PR has a good title :heavy_check_mark:

## (Changes Required) 

- [ ] Please I just noticed that you first section is not rendered properly.

## Your design

crr17

- [ ] Take note of other changes required from the inline comments as well 👍 

 
## [Optional] suggestions:
 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
**Cheers and Happy coding! :clap::clap::clap:**
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear
Please, remember to tag @oyepriscilla in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by monicah79** [comment]
Thanks for the feedback! but when i add ul its disorganising my work ,,,how
do i add ul ,,,outside a div or inside

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @monicah79 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights :heavy_check_mark: 
:white_check_mark:  No Linters error
:white_check_mark: Descriptive pull request
:white_check_mark: Great working on the set up the project 
:white_check_mark: Good commit messages 
:white_check_mark: Professional README file

Congratulations-Memes-FEATURED

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by OyePriscilla** [comment]
Hi @monicah79, I apologise for the late response. I am glad your work is approved; be determined and keep moving. You can do it 👍 👍

**Pull request merged**